core/src/test/scala/org/apache/spark/storage/MemoryStoreSuite.scala

```diff
@@ -60,7 +60,7 @@ class MemoryStoreSuite
     SizeEstimator invokePrivate initialize()
   }
 
-  def makeMemoryStore(maxMem: Long): (MemoryStore, BlockInfoManager) = {
+  def makeMemoryStore(maxMem: Long, conf: SparkConf = conf): (MemoryStore, BlockInfoManager) = {
```

Comment:
Oh, that's confusing because of the shadowing. The arg and member have the same name. Please name this arg differently if you do this.